crates/store/re_video/src/decode/av1.rs

```diff
@@ -112,163 +114,178 @@ fn output_picture(picture: &dav1d::Picture, on_output: &(dyn Fn(Result<Frame>) +
     // TODO(jan): support other parameters?
     // What do these even do:
     // - matrix_coefficients
-    // - color_range
-    // - color_primaries
     // - transfer_characteristics
 
-    let frame = Frame {
-        data: match picture.pixel_layout() {
-            PixelLayout::I400 => i400_to_rgba(picture),
-            PixelLayout::I420 => i420_to_rgba(picture),
-            PixelLayout::I422 => i422_to_rgba(picture),
-            PixelLayout::I444 => i444_to_rgba(picture),
+    let data = {
+        re_tracing::profile_scope!("copy_picture_data");
+
+        match picture.pixel_layout() {
+            PixelLayout::I400 => picture.plane(PlanarImageComponent::Y).to_vec(),
+            PixelLayout::I420 | PixelLayout::I422 | PixelLayout::I444 => {
+                // TODO(#7594): If `picture.bit_depth()` isn't 8 we have a problem:
+                // We can't handle high bit depths yet and the YUV converter at the other side
+                // bases its opinion on what an acceptable number of incoming bytes is on this.
+                // So we just clamp to that expectation, ignoring `picture.stride(PlanarImageComponent::Y)` & friends.
+                // Note that `bit_depth` is either 8 or 16, which is semi-independent `bits_per_component` (which is None/8/10/12).
+                if picture.bit_depth() != 8 {
+                    re_log::warn_once!(
+                        "Video uses unsupported bit depth larger than 8 bit per color component which is currently unsupported."
```

Comment:
`bit_depth` is a weird one. it's either 8 or 16 because that's the values the library generates. but then there's the component bit count which is 8/10/12 or none (wtf -.-). Obviously there's a correlation but it's not spelled out in docs :sigh: ... but yeah let's just output it